app/javascript/components/student/editor/TestsList.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { TestSummary } from './TestSummary'
+import { TestStatus, Test } from './TestRunSummary'
+
+function Overview({ tests }: { tests: Test[] }) {
+  const passed = tests.reduce(function (total, test) {
+    if (test.status === TestStatus.PASS) {
+      total += 1
```

Comment:
It's common, if this is what you do inside the reduce, to instead use .filter.length